src/setupTests.ts

```diff
@@ -3,5 +3,11 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import '@testing-library/jest-dom/extend-expect'
+import secondsToMilliseconds from 'date-fns/secondsToMilliseconds'
 
 jest.mock('no-scroll', () => ({ on: jest.fn(), off: jest.fn() }))
+
+beforeEach(() => {
+  // Mock jest's "Today"
+  Date.now = jest.fn(() => secondsToMilliseconds(1638350762)) // 01.12.2021
+})
```

Comment:
thought: The only thing that worries me about putting this here is that it might catch someone by surprise not knowing why their test is acting a certain way. thoughts @gautierlabarre  ?

Reply:
yeah, in some tests, it was actually mocked with another date so it was overrided, a bit useless I guess but I like the idea of having a "today" mocked anyway to avoid the "oh the tests worked last week but not anymore due to date" issue when you forget to mock it